### PR TITLE
Redirection for REPRODUCE-ME repository

### PR DESCRIPTION
**Pull request by Sheeba-Samuel**
Added redirection for REPRODUCE-ME repository and ontology (https://sheeba-samuel.github.io/REPRODUCE-ME/)

### CODE REVIEW

**Comment by dgarijo** [comment]
The URI of the ontology is http://w3id.org/reproduceme# (e.g., http://w3id.org/reproduceme#name). However, the w3id for the ontology is http://w3id.org/reproduceme/ontology.
I recommend doing the content negotiation on http://w3id.org/reproduceme instead, otherwise applications will be confused.

**Comment by Sheeba-Samuel** [comment]
Thanks @dgarijo for the quick reply. As per the recommendation, I have added the content negotiation on http://w3id.org/reproduceme.

File: reproduceme/.htaccess [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^$ https://sheeba-samuel.github.io/REPRODUCE-ME/doc/index-en.html [R=303,L]
+
+# Rewrite rule to serve JSON-LD content from the REPRODUCE-ME vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} application/ld+json
```
Comment (dgarijo) [inline reproduceme/.htaccess]: This should be application/ld\+json (sorry, it's a typo in WIDOCO that will be addressed in the next release)

**Comment by dgarijo** [comment]
I added a small comment with a typo. Please fix that and I will accept it

**Comment by Sheeba-Samuel** [comment]
Thanks @dgarijo for pointing out the typo.

**Pull request merged**